Document/0x05f-Testing-Local-Authentication.md

```diff
@@ -280,4 +281,3 @@ Patch the app or use runtime instrumentation to bypass fingerprint authenticatio
 #### Request App Permissions
 
 - Runtime Permissions - https://developer.android.com/training/permissions/requesting
-- Samsung Pass Developer Guide - https://developer.samsung.com/galaxy/pass/guide

```

Comment:
We could at least mention Samsung pass in the references https://developer.samsung.com/tech-insights/pass/using-samsung-pass. 
Any reason to remove it completely?

Reply:
It was only mentioned in a bad way, and we try not to do that. The URL was also no longer valid. Maybe it should be added in a different section as a general reference for a biometrics sdk ?

Reply:
Ok, then it makes sense to remove it. Let's leave it for now, if we start with this the we should also look into and mention other SDKs ;-)

Reply:
The Samsung Pass SDK has the same issue in Android as using pureluy LAContext in iOS: it will allow for an easy patchable bypass of the touchevent. Therefore, we should mention it as examples of how not to implement biometrics...